CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/1.9.1...HEAD)
+* **Flink: improve Cassandra lineage metadata** (https://github.com/OpenLineage/OpenLineage/pull/2479) [@HuangZhenQiu](https://github.com/HuangZhenQiu)
```

Comment:
Would it be worth adding Cassandra naming convention to:
 - https://openlineage.io/docs/spec/naming
 - https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md

Sorry for having same definition in both places.

Reply:
Updated in https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md. But I am not able to find the source doc file for https://openlineage.io/docs/spec/naming.

Reply:
https://github.com/OpenLineage/docs/blob/main/docs/spec/naming.md